src/match.c

```diff
@@ -2784,6 +2801,21 @@ void PlayerReady(qbool startIdlebot)
 			G_bprint(2, "All players ready\n");
 		}
 
+		if (cvar("k_on_start_f_modified"))
+		{
+			stuffcmd(self, "say f_modified\n");
+		}
+
+		if (cvar("k_on_start_f_ruleset"))
+		{
+			stuffcmd(self, "say f_ruleset\n");
+		}
+
+		if (cvar("k_on_start_f_version"))
```

Comment:
third time's the charm?

Reply:
oh, I'm blind :D

Reply:
The `f_ruleset` check truncates the version string, so `f_version` is unfortunately needed as well :(